<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logoView</class>
 <widget class="QMainWindow" name="logoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../img.qrc">
    <normaloff>:/img/fc81f9531389b9246ecb09838cdb851.jpg</normaloff>:/img/fc81f9531389b9246ecb09838cdb851.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1300</width>
      <height>730</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1300</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1300</width>
      <height>16777215</height>
     </size>
    </property>
    <widget class="QLineEdit" name="keywordEdit">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>66</y>
       <width>481</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>120</y>
       <width>876</width>
       <height>500</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>876</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>876</width>
       <height>500</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="queryButton">
     <property name="geometry">
      <rect>
       <x>495</x>
       <y>66</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: rgb(90, 116, 144);
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="homeButton">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>51</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>51</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/4a154c900c9b2cdcfc7e5ddb8c535cf.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>610</y>
       <width>550</width>
       <height>100</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>550</width>
       <height>100</height>
      </size>
     </property>
     <widget class="QLineEdit" name="deleteEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>输入待删除的id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(168, 90, 78);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>确认删除</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>610</y>
       <width>350</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>81</height>
      </size>
     </property>
     <widget class="QPushButton" name="pageDownButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>下一页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pageUpButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>上一页</string>
      </property>
     </widget>
     <widget class="QLabel" name="pageLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>22</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>pageNow/totalPage</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>110</y>
       <width>361</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/retouch_2024122911010896.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1300</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/926386dea79cf2ea36dcb89162ddac4.jpg);</string>
    </property>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections/>
</ui>
